src/components/cards/SupportCard.tsx

```diff
@@ -17,11 +18,13 @@ export const SupportCard = ({ img, alt, name, date }: Props) => {
           <p className="md:text-xl">{date}</p>
         </div>
       </div>
-      <img
-        src={calendarIcon}
-        alt="kalendář"
-        className="absolute w-5 h-5 right-4 bottom-4"
-      />
+      <a href={link} target="_blank">
```

Comment:
as it's a link of a 3rd party, I'd suggest to add this attribute to it `rel="noopener noreferrer"`